gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
By dev machine, I really meant for those folks that might use VS code as their debugger, these tagalong files are helpful to include